### PR TITLE
Fix/image csp for all of GitHub

### PR DESCRIPTION
**Pull request by EstebanDalelR**
## Description
Some instances of CI/CD images were not displayed. 
Relaxed the CSP rules to allow all of github

## Type of change

- Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 - Fixed an error where the command would break even when results were fetched
 - Fixed some instances of the get help on Slack button not working
 - Will select whole file if no selection is made
+- Fixed case where empty PR body or empty comment would break the app
```
Comment (baristaGeek) [inline CHANGELOG.md:15]: This is not part of this PR

**Review by baristaGeek — CHANGES_REQUESTED** [review]
Remove the mentioned line. Else, looks good.

File: CHANGELOG.md (hunk 1 above)
Comment (EstebanDalelR) [reply CHANGELOG.md]: Solved.

**Review by baristaGeek — APPROVED** [review]
